jmclient/jmclient/blockchaininterface.py

```diff
@@ -31,6 +31,10 @@ def is_address_imported(self, addr):
         except JsonRpcError:
             return len(self.rpc('getaddressinfo', [addr])['labels']) > 0
 
+    @abc.abstractmethod
+    def is_address_labeled(self, utxo, walletname):
```

Comment:
There's an argument that this may not be needed for some interfaces.

But, one can always address that with dummy functions, so this is fine.